### DISCUSSION TITLE
Device Tab goes to predefined Device Manager Page e.g. "Klipper" / Moonraker / Octoprint / RRF

### DISCUSSION THREAD

**Discussion by sylanc3** [OrcaSlicer/OrcaSlicer#65] [Ideas]
Hello SoftFever,
I'm a Newbe on Git. But I had an Idea.
It would be nice to have an implementation to BLS SoftFever, that when a Printer other than BL is used, that the "Device" tab is showing Klipper / "Moonraker", what do you think?
![2022-12-27 11_31_39-Untitled](https://user-images.githubusercontent.com/76587995/209653732-642a570f-ab6a-4e23-bbf2-7c011641457c.png)

**Comment by netweaver1970** [comment]
great idea, it would 'fit' in the all-in-program approach.

**Comment by SoftFever** [comment]
Thanks for the suggestion.
I prefer to keep it "neutral" as there are octoprint/RRF users as well. Or any other printer with a web UI :)
